react_app/src/hooks/useSearches.tsx

```diff
@@ -3,13 +3,14 @@ import { notification } from 'antd';
 import { useLocalStorage } from '@rehooks/local-storage';
 import { Search } from '../types/Search';
 import { Namespace } from '../types/Namespace';
-import { usePrevious } from './usePrevious';
 import { urlFor } from '../utils/urlFor';
 import { parseFASTA } from '../utils/handleFASTA';
 
-async function fetchStatuses() {
+async function fetchStatuses(ids: Search['id'][]) {
```

Comment:
Added support for the new `jobIds` required API param

Reply:
see https://lb.genesearch.production.svc.spin.nersc.org/docs/#operation/getJobStatuses